### PR TITLE
Import reload

### PR DESCRIPTION
**Pull request by jklenzing**
Summary of changes
- Updated import of reload statements now that python 3.3 has reached end of life (consistent with changes discussed in updates to utils #164)
- Reordered library imports for testing files as in other pull requests
- Deleted commented tests that have been moved to other testing files
- Deleted comments in setup from update of pyglow
- General pep8 scrub of changed files

### CODE REVIEW

File: pysat/tests/test_instrument.py [hunk 1]
```diff
@@ -1,24 +1,24 @@
...
-        reload = importlib.reload
+    from importlib import reload as re_load
+else:
+    re_load = reload
 
 
 class TestBasics():
     def setup(self):
-        reload(pysat.instruments.pysat_testing)
+        re_load(pysat.instruments.pysat_testing)
```
Comment (aburrell) [inline pysat/tests/test_instrument.py]: @rstoneback or @jklenzing , why is this necessary for a clean testing set up?

File: setup.py [hunk 2]
```diff
@@ -25,11 +25,6 @@
                     'pyEphem', 'requests', 'pysatCDF', 'apexpy', 'aacgmv2',
                     'pysatMagVect', 'madrigalWeb', 'h5py', 'PyForecastTools']
 
-# keep pyglow out of requirements until pip installable
-# if sys.version_info[0] < 3:
-#     # TODO Remove when pyglow works in python 3
-#     install_requires.append('pyglow')
-
```
Comment (aburrell) [inline setup.py]: Yay!

File: pysat/tests/test_instrument.py (hunk 1 above)
Comment (jklenzing) [reply pysat/tests/test_instrument.py]: I'm not sure.  @rstoneback?

File: setup.py (hunk 2 above)
Comment (jklenzing) [reply setup.py]: Forgot to add this to the list of changes.  :D

File: pysat/tests/test_instrument.py (hunk 1 above)
Comment (jklenzing) [reply pysat/tests/test_instrument.py]: Adding to agenda for today's meeting.  This has been one of my questions for a while.

File: setup.py (hunk 2 above)
Comment (jklenzing) [reply setup.py]: Wait, I think this is implemented wrong.  I think I should have just removed the if statement, not the install_requires bit.  @rstoneback?

Comment (aburrell) [reply setup.py]: Ah, I think so!  I think you just remove the comments?

Comment (jklenzing) [reply setup.py]: Or just add it to the list above, since we don't need to append.  Updated readthedocs truncation as well.

File: pysat/tests/test_instrument.py (hunk 1 above)
Comment (rstoneback) [reply pysat/tests/test_instrument.py]: I mess with the module in a couple tests, lines 210 and 222.

File: pysat/tests/test_instrument.py [hunk 3]
```diff
@@ -189,19 +189,6 @@ def test_instrument_init(self):
...
-#         assert ((yr == 2009) & (doy == 1))
-#
-#     def test_getyrdoy_leap_year(self):
-#         """Test the date to year, day of year code functionality
-#         (leap_year)"""
-#         date = pds.datetime(2008,12,31)
-#         yr, doy = pysat.utils.getyrdoy(date)
-#         assert ((yr == 2008) & (doy == 366))
-
     def test_custom_instrument_load(self):
```
Comment (rstoneback) [inline pysat/tests/test_instrument.py]: I think this method and the two below it may need to be in their own class. This whole suite of tests is called for xarray and other configs yet the modules are fixed to pysat_testing, so they aren't being run correctly for other situations. We can also move the reload to only handle these tests.

Comment (rstoneback) [reply pysat/tests/test_instrument.py]: Ideally the tests should be modified to accept the other testing instruments. Anytime I leave a gap in testing there is sure to be a bug.

File: pysat/tests/test_instrument.py [hunk 4]
```diff
@@ -559,15 +546,16 @@ def test_incorrect_creation_empty_instrument_object(self):
...
+            pass
         Dummy.name = 'help'
 
         temp = pysat.Instrument(inst_module=Dummy)
 
 
 class TestBasicsXarray(TestBasics):
     def setup(self):
-        reload(pysat.instruments.pysat_testing)
+        re_load(pysat.instruments.pysat_testing)
```
Comment (rstoneback) [inline pysat/tests/test_instrument.py]: All of these re_loads aren't really correct due to my previous generalization that wasn't quite complete. To be consistent, it should reallybe a reload of pysat_testing_xarray, but then that module doesn't get tested properly as noted in comment above.

File: setup.py (hunk 2 above)
Comment (rstoneback) [reply setup.py]: Updated version looks good to me.

File: pysat/tests/test_instrument.py (hunk 1 above)
Comment (aburrell) [reply pysat/tests/test_instrument.py]: Let's put some comments above this to that effect, so that people looking for code examples will know why this is done.

File: pysat/tests/test_instrument.py (hunk 4 above)
Comment (rstoneback) [reply pysat/tests/test_instrument.py]: Enhancement noted by issue #182

File: pysat/tests/test_instrument.py (hunk 3 above)
Comment (rstoneback) [reply pysat/tests/test_instrument.py]: enhancement noted in issue #182

File: pysat/tests/test_instrument.py (hunk 1 above)
Comment (rstoneback) [reply pysat/tests/test_instrument.py]: That is reasonable. Issue created to remind us to more completely solve in the future.

**Comment by jklenzing** [comment]
including latest updates from develop
